variables.tf

```diff
@@ -744,3 +744,13 @@ variable "variance_offset_mins" {
 variable "verify_rate_limit_secs" {
   description = "Time in seconds a user must wait before attempting to verify a one-time upload code"
 }
+
+variable "self_isolation_notices_enabled" {
+  description = "Flag (string, required by SSM) to enable/disable self-isolation notices"
+  default     = "false"
+}
+
+variable "self_isolation_notices_url" {
+  description = "Self isolation notices target url. Empty by default, conditioned by self_isolation_notices_enabled"
+  default     = "NA"
```

Comment:
Should this be empty string (`""`) as per the description?

Reply:
It does not work with empty string. I tried and AWS complains:
```
Error: error creating SSM parameter: ValidationException: 1 validation error detected: Value '' at 'value' failed to satisfy constraint: Member must have length greater than or equal to 1.
        status code: 400, request id: 1846ab1b-9b4c-41bb-9213-83c02ab9128f

  on ../../covid-green-infra/self-isolation-notices-common.tf line 7, in resource "aws_ssm_parameter" "self_isolation_notices_url":
   7: resource "aws_ssm_parameter" "self_isolation_notices_url" {
```